docker-compose.yml

```diff
@@ -140,6 +143,16 @@ services:
             configdb:
                 condition: service_healthy
 
+    ocs_frontend:
+        image: observatorycontrolsystem/ocs-example-frontend:v0.0.1
```

Comment:
Why is there a v in front of the version number?

Reply:
I was following the recommendation for tag names that github displays in the page for creating a new release. But I see it's not consistent with the other ocs repositories... should I remove the v?

Reply:
Yeah I think we should stay consistent and not have the v